docs/source/new-features/index.rst

```diff
@@ -46,6 +67,82 @@ Find out about the details of major features, changes, and deprecations below.
+                    <div class="body">
+                        When performing training of Deep Learning models with PyTorch Lightning or Keras, model checkpoint saving 
+                        is enabled, allowing for state storage during long-running training events and the ability to resume if 
+                        an issue is encountered during training. 
+                    </div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Mistral AI added as an MLflow Deployments Provider
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/mistral-ai-logo.svg" alt="Mistral AI" style="max-height: 5rem"></img>
+                    <div class="body">
+                        The <a href="../llms/deployments/index.html">MLflow Deployments Server</a> can now 
+                        accept <a href="https://mistral.ai/">Mistral AI</a> endpoints. Give their models a try today! 
+                    </div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        Keras 3 is now supported in MLflow
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/keras-logo.svg" alt="Keras"></img>
+                    <div class="body">
+                        You can now log and deploy models in the new <a href="https://keras.io/keras_3/">Keras 3 format</a>, allowing you 
+                        to work with TensorFlow, Torch, or JAX models with a new high-level, easy-to-use suite of APIs.
+                    </div>
+                    <div class="tag">
+                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-card">
+                <div class="content-container">
+                    <div class="header">
+                        MLflow now has support for OpenAI SDK 1.x
+                    </div>
+                    <img class="card-image" src="../_static/images/logos/openai-logo.svg" alt="OpenAI" style="max-height: 5rem"></img>
+                    <div class="body">
+                        We've updated flavors that interact with the OpenAI SDK, bringing full support for the API changes with the 1.x release.
```

Comment:
Do we have any good reference for the required code update for users? I guess it's only `task=openai.ChatCompletion` -> `task=openai.chat.Completion` and we've updated code examples in our doc ([here](https://github.com/mlflow/mlflow/pull/11123/files#diff-27092883ef9a4a81ab9b5ea59442ab48dd314babd61428270948fcfae0a80f28)). Maybe we can add a link to one of them as quick reference?